File: src/main/java/cz/stasimek/fakturaceeasypeasy/config/StaticResourcesConfiguration.java
package cz.stasimek.fakturaceeasypeasy.config;

import org.springframework.beans.factory.annotation.Value;
import org.springframework.context.annotation.Configuration;
import org.springframework.web.servlet.config.annotation.ResourceHandlerRegistry;
import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;

@Configuration
public class StaticResourcesConfiguration implements WebMvcConfigurer {

	@Value("${app.static-resources-directory}")
	String staticResourcesDirectory;

	@Override
	public void addResourceHandlers(ResourceHandlerRegistry registry) {
		registry.addResourceHandler("/static/**").addResourceLocations(staticResourcesDirectory);
	}
}

File: src/main/kotlin/cz/stasimek/fakturaceeasypeasy/config/StaticResourcesConfiguration.kt
package cz.stasimek.fakturaceeasypeasy.config

import org.springframework.beans.factory.annotation.Value
import org.springframework.context.annotation.Configuration
import org.springframework.web.servlet.config.annotation.ResourceHandlerRegistry
import org.springframework.web.servlet.config.annotation.WebMvcConfigurer

@Configuration
class StaticResourcesConfiguration(
    @Value("\${app.static-resources-directory}") private val staticResourcesDirectory: String
) : WebMvcConfigurer {

    override fun addResourceHandlers(registry: ResourceHandlerRegistry) {
        registry.addResourceHandler("/static/**").addResourceLocations(staticResourcesDirectory)
    }
}
